helix-term/src/commands.rs

```diff
@@ -228,13 +228,14 @@ impl Command {
         goto_definition, "Goto definition",
         goto_type_definition, "Goto type definition",
         goto_implementation, "Goto implementation",
-        goto_file_start, "Goto file start",
+        goto_file_start, "Goto file start/line",
```

Comment:
Would it be better if we just keep this but only use `G`?

Reply:
I added `gg` since it's easier to double tap than to hold shift and tap `g`. YMMV of course but I think it's nice to have both.